script.js

```diff
@@ -0,0 +1,83 @@
+function runPizza() {
+  const vegetarian = "Vegetarian Pizza"
+  const hawaiian = "Hawaiian Pizza"
+  const pepperoni = "Pepperoni Pizza"
+
+  const pizzaPrice = 80
+
+  //Put your Javscript code here:
+  // functions
+  function checkOrder (name){
+    const type = name 
+    let status = true
+    if (name === pepperoni || name === hawaiian || name === vegetarian){
+      alert("good choice")
+      status = true
+    } 
+    else{
+      alert("sorry try again")
+      status = false
+    } 
+    return status
+    }
+  function cost (num, pricr){
```

Comment:
A good thing to do is doing the functions in the new way (ES6) instead of this old way. It's good to get the hang of it from the start once you go out and work as developers. Ex
```
const checkOrder = name => {
  // do something
}
```